Source/Scene/PostProcessStageCollection.js

```diff
@@ -0,0 +1,682 @@
+            }
+        },
+        /**
+         * A post-process stage for Fast Approximate Anti-aliasing.
+         * <p>
+         * When enabled, this stage will execute after all others.
+         * </p>
+         *
+         * @memberof PostProcessStageCollection.prototype
+         * @type {PostProcessStage}
+         * @readonly
+         */
+        fxaa : {
+            get : function() {
+                return this._fxaa;
+            }
+        },
+        /**
+         * A post-process stage that applies Horizon-based Ambient Occlusion (HBAO) to the input texture.
+         * <p>
+         * Ambient occlusion simulates shadows from ambient light. These shadows would always be present when the
+         * surface receives light and regardless of the light's position.
+         * </p>
+         * <p>
+         * The uniforms have the following properties: <code>intensity</code>, <code>bias</code>, <code>lengthCap</code>,
+         * <code>stepSize</code>, <code>frustumLength</code>, <code>ambientOcclusionOnly</code>,
+         * <code>delta</code>, <code>sigma</code>, and <code>kernelSize</code>.
+         * </p>
+         * <p>
+         * <code>intensity</code> is a scalar value used to lighten or darken the shadows exponentially. Higher values make the shadows darker. The default value is <code>3.0</code>.
+         *
+         * <code>bias</code> is a scalar value representing an angle in radians. If the dot product between the normal of the sample and the vector to the camera is less than this value,
+         * sampling stops in the current direction. This is used to remove shadows from near planar edges. The default value is <code>0.1</code>.
+         *
+         * <code>lengthCap</code> is a scalar value representing a length in meters. If the distance from the current sample to first sample is greater than this value,
+         * sampling stops in the current direction. The default value is <code>0.26</code>.
+         *
+         * <code>stepSize</code> is a scalar value indicating the distance to the next texel sample in the current direction. The default value is <code>1.95</code>.
+         *
+         * <code>frustumLength</code> is a scalar value in meters. If the current fragment has a distance from the camera greater than this value, ambient occlusion is not computed for the fragment.
+         * The default value is <code>1000.0</code>.
+         *
+         * <code>ambientOcclusionOnly</code> is a boolean value. When <code>true</code>, only the shadows generated are written to the output. When <code>false</code>, the input texture is modulated
+         * with the ambient occlusion. This is a useful debug option for seeing the effects of changing the uniform values. The default value is <code>false</code>.
+         * </p>
+         * <p>
+         * <code>delta</code>, <code>sigma</code>, and <code>kernelSize</code> are the same properties as {@link PostProcessStageLibrary#createBlurStage}.
```

Comment:
These properties look a bit crowded in the documentation. Maybe a bulleted list would help.
<img width="540" alt="list" src="https://user-images.githubusercontent.com/915398/38814988-dbb9805e-4160-11e8-91b4-3810fbbacca4.PNG">

Reply:
Same comment for bloom and any of the stages with multiple uniforms in `PostProcessStageLibrary` (wherever it makes sense)..